docs/UserGuide.md

```diff
@@ -155,7 +159,7 @@ Examples:
 * `add_contact n/John Doe p/98765432 e/johndtr@example.com a/John street, block 123, #01-01`
```

Comment:
A spacing needed between `</box>` and the next line, otherwise, formatting gets truncated.